deepspeed/runtime/zero/config.py

```diff
@@ -339,7 +340,10 @@ class DeepSpeedZeroConfig(DeepSpeedConfigModel):
     """
     Override nn.Module apply function, for Stage 3.
     """
-
+    autotp_size: int = Field(0, ge=0, new_param="autotp_size")
```

Comment:
Why is `autotp_size` defined as a subfield of zero, instead of a top-level field in ds_config? Is there a dependency on zero logic?